pnda_env_example.yaml

```diff
@@ -194,6 +194,10 @@ parameter_defaults:
   #
   #hypervisor_count: 30
 
+  # Network interface to be used for Kafka nodes for ingestion.
+  kafka_grain_pnda_internal_network: eth0
```

Comment:
could you keep this align with the pnda-cli definition please? as this is not kafka specific, so should be pnda_internal_network instead.